routes/employees.js

```diff
@@ -1,17 +1,20 @@
 const express = require('express')
 const router = express.Router()
 const employeesController = require('../controllers/employees')
```

Comment:
i think we should go ahead and create the same file structure like you did on salaries routes to make the code base uniformed.